cypress/README.md

```diff
@@ -0,0 +1,110 @@
+# Cypress Tests
+
+The FFG Star Wars Enhancements module uses [Cypress](https://www.cypress.io/) to run its tests.
+
+## Overview
+
+Cypress automates front end testing by emulating user interactions within a browser.
+
+## Testing locally
+
+> :warning: Cypress tests run against FoundryVTT's default port of 30000 by default. The tests take actions that may be harmful to any real games running in that environment. See Configuration below to change the default.
+
+Cypress itself does not orchestrate launching FoundryVTT. By default, it expects a fresh FoundryVTT instance to be running on `http://localhost:30000`.
+
+To run Cypress [headless](https://en.wikipedia.org/wiki/Headless_browser) from the commandline:
+
+```shell
+# Install dependencies
+npm install
+
+# Run Cypress
+npx cypress run
+```
+
+## Developing locally
```

Comment:
Developing... tests locally?

Reply:
This should probably be Testing locally.